Atomic/syscontainers.py

```diff
@@ -197,7 +201,11 @@ def _checkout_system_container(self, repo, name, img, deployment, upgrade, value
         imagebranch = SystemContainers._get_ostree_image_branch(img)
 
         destination = destination or "%s/%s.%d" % (self._get_system_checkout_path(), name, deployment)
-        exports = os.path.join(destination, "rootfs/exports")
+        if self.args.remote:
```

Comment:
instead of using `self.args.remote` in different places, I would instead set `self.remote` as:

`self.remote = self._resolve_remote_path(self.args.remote)`

For now the `_resolve_remote_path` can just resolve symlinks, but we will change later to resolve images exploded from the Atomic Registry.

You can then drop the control in `update_system_container` as the rootfs might change as well, if the `_resolve_remote_path` gives a different result (e.g. the symlink points to a different version).